enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/mock/MockCustomActivationProvider.java

```diff
@@ -17,13 +17,13 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package com.wultra.app.enrollmentserver.mock;
+package com.wultra.app.onboardingserver.mock;
```

Comment:
Oops, this is not working. `CustomActivationProvider` have to be registered in enrollment spring context, but this implementation depends on `enrollment-onboarding`.